src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -311,7 +349,12 @@ public DatasetTerm toDatasetTerm(Dataset dataset) {
     }
 
     if (Objects.nonNull(dataset.getDataUse())) {
-      term.setDataUse(ontologyService.translateDataUseSummary(dataset.getDataUse()));
+      DataUseSummary summary = ontologyService.translateDataUseSummary(dataset.getDataUse());
+      if (summary != null) {
+        term.setDataUse(summary);
+      } else {
+        logWarn("No data use summary for dataset id: %d".formatted(dataset.getDataSetId()));
+      }
```

Comment:
Miscellaneous refactoring effort - Ontology can choke on invalid data use translations, so this allows us to follow up on those datasets that need attention. Will follow up with a new ticket to address this in Ontology.